pkg/ingestor/api/api.go

```diff
@@ -52,6 +56,59 @@ func NewIngestorAPI(cfg *config.KubehoundConfig, puller puller.DataPuller, notif
 	}
 }
 
+// RehydrateLatest is just a GRPC wrapper around the Ingest method from the API package
+func (g *IngestorAPI) RehydrateLatest(ctx context.Context) ([]*grpc.IngestedCluster, error) {
```

Comment:
From on design PoV, I think it would be better to have this function just do:
- call a function that get []string of the list of the latest dump 
- loop on it to call ingest()

This would need to move most of this to a funciton like `LatestDumpByCluster`, maybe in the puller package?

Reply:
It is already kind of the case (listing the files, and then iterating over). 
The only case I see where it could be useful would be if we want to rehydrate specific clusters, but it will require to add news entries more work which is not needed for the moment. This new function is mainly to "reload" when the graph reboots.

Reply:
yes, I'm advocating for a split of the function in 2:
- part 1 being the "extract the latest clusters"
- part 2: call the api based on the results of that